### ISSUE TITLE
Mysterious semicolons in docs `core/howto/quotes`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4944]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4944|
|Type|enhancement|
|Created|2011-03-14 08:59:30Z|
|Branch|https://github.com/twisted/twisted/tree/remove-semicolons-from-quotes-docs-4944|

From [#4865](https://github.com/twisted/twisted/issues/4865), tenth says:

> I noticed that there are mysterious semicolons after the second and third Python code blocks... They are present in the original Lore version of the docs, but they look like typos to me. Unless they have some purpose, we should probably remove those as well?

These probably should be removed, but as they are not really a problem for the conversion to Sphinx, I'm creating a new ticket.

Attachments:

* [quotes-semicolon.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/82a/82a174bbd3ef0a3ab292d5a9c59acd182e531d27/77fad04194565215a6b083f4376ef32ce5ddd37f.patch) (1064 bytes) - added by thijs on 2011-07-01 15:51:21Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4944 4944
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__branches_remove_semicolons_from_quotes_docs_4944 branches/remove-semicolons-from-quotes-docs-4944
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1300093170000000 1300093170000000
changetime__1309846456000000 1309846456000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

(In [32215]) Branching to 'remove-semicolons-from-quotes-docs-4944'

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli commented|
|-|-|

(In [32216]) Apply quotes-semicolon.patch

refs [#4944](https://github.com/twisted/twisted/issues/4944)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli set owner to @thijstriemstra|
|-|-|

Thanks for working on this thijs!  [doc buildbot](http://buildbot.twistedmatrix.com/builders/documentation/builds/1372) looks fine.  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Thanks for the review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32227]) Merge remove-semicolons-from-quotes-docs-4944: Remove semicolons from the TwistedQuotes directory structure overview.

Author: thijs
Reviewer: cyli
Fixes: [#4944](https://github.com/twisted/twisted/issues/4944)
